browser/ui/tabs/brave_tab_menu_model.cc

```diff
@@ -158,6 +158,9 @@ void BraveTabMenuModel::BuildItemsForSplitView(
 
   if (brave::IsTabsTiled(browser, indices)) {
     InsertItemWithStringIdAt(++index, CommandBreakTile, IDS_IDC_BREAK_TILE);
```

Comment:
Curious why break tile entry doesn't have same policy if tiles are selected but non-tile tab is active?

Reply:
Break tile works for all selected tabs in split view. I think that's okay because we can see which tabs are tiled by the tile background. But in case of the tabs position, I thought it'd be better not to support switching tabs in background as users can't see the contents.

Reply:
But it's open to support background tabs, for now I was just focusing on fixing the crash.